### ISSUE TITLE
Play button doesnot disappear when the video is opened twice

### ISSUE THREAD

**Issue by skshetry** [owncloud/files_mediaviewer#74]
### How to reproduce:
1. Open a folder (preferably via direct link/hard reload).
2. Open the video file in that folder and play.
3. Click anywhere outside the viewer (will stop the video and viewer will disappear).
4. Open a video file in the same folder (any one) and play. 

### Expected Outcome:
The video gets played, and the play button disappears. 

### Actual Outcome:
1. The play button does not disappear.
2. The controls' bar also shows paused state.
3. However, the video does get played.  And, clicking on pause/play button on the video overlay and the controls works, only the state does not change.

### Affected Versions:
1. `RC2`
2. `release-1.0.0`

I tried on the `master` as well, but could not reproduce.

### Screenshot
![output](https://user-images.githubusercontent.com/18718008/57056305-4eeaa800-6cc1-11e9-9774-0c1aa4b324e8.gif)

The following is a `vuex` export of `release-1.0.0` when the second video is running with play button superimposed over the video in above gif. Thought, it could be helpful. 

```
{
  "activeIndex":2,
  "maxIndex":4,
  "activeMediaItem":{
    "id":"2147565623",
    "path":"/",
    "name":"random_video.mp4",
    "mtime":1556613657000,
    "etag":"a4af0ff2532dc11dff577a92db06323f",
    "size":40836280,
    "mimetype":"video/mp4",
    "permissions":27,
    "mountType":null,
    "commentsUnread":0,
    "type":"file"
  },
  "activeHTMLElement":{
    "0":"[object HTMLVideoElement]",
    "length":1,
    "prevObject":{
      "0":"[object HTMLDocument]",
      "context":"[object HTMLDocument]",
      "length":1
    },
    "context":"[object HTMLDocument]",
    "selector":".swiper-slide-active .viewer__media"
  },
  "isLoading":false,
  "isInTransition":false,
  "video":{
    "isPaused":true,
    "isMuted":false,
    "isFullscreen":false
  }
}
```

Here's the vuex state from devtools:
![Screenshot from 2019-05-02 09-55-53](https://user-images.githubusercontent.com/18718008/57056206-97ee2c80-6cc0-11e9-8372-2010b00e6d40.png)

I can provide more information if required.

**Comment by skshetry** [comment]
`git bisect` shows the bug seems to have been introduced with #71.

**Comment by skshetry** [comment]
Sometimes, the "Play" button does not disappear even when just opened once.
Scrubber does not show up if the `Play` button does not disappear.

**Comment by skshetry** [comment]
And, if it appears in second time, the scrubber does appear. But, the scrubber doesnot change position when clicked on the different position on the scrubber, though the video changes.

**Comment by skshetry** [comment]
Fixed by #84
